x-pack/plugin/identity-provider/src/main/java/org/elasticsearch/xpack/idp/IdentityProviderPlugin.java

```diff
@@ -80,12 +84,22 @@
         }
 
         SamlInit.initialize();
-        final SamlIdentityProvider idp = SamlIdentityProvider.builder().fromSettings(environment).build();
         final SamlServiceProviderIndex index = new SamlServiceProviderIndex(client, clusterService);
+
+        // TODO
```

Comment:
is this TODO about possibly exposing sp defaults in IDP settings?

Reply:
Yes, these should definitely be configurable, and this isn't the place to set them, but I wanted to avoid Yak shaving here.